packages/xod-project/src/attachment.js

```diff
@@ -11,14 +11,21 @@ export const createAttachment = def(
   })
 );
 
-export const createImplAttachment = def(
-  'createImplAttachment :: Source -> Attachment',
-  content => createAttachment(IMPL_FILENAME, 'utf-8', content)
+export const createAttachmentManagedByMarker = def(
+  'createAttachmentManagedByMarker :: PatchPath -> String -> Attachment',
+  (markerName, content) =>
+    createAttachment(
+      // TODO: what about unknown filenames?
```

Comment:
I think it’s a bug. Pick from `MANAGED_ATTACHMENT_FILENAMES` with Maybe, explode with assertion